### PR TITLE
{bio}[fosscuda/2020b] AlphaFold v2.0.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3302.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/7715dc604e16aeab8c9b893aaeba18c4 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 3 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.5
See https://gist.github.com/43825459c05812acbf25ff445c88e577 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in total)
b-an04.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.5
See https://gist.github.com/234e349caf7c56f92b9b1539008953c7 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jwvis00.juwels - Linux centos linux 8.4.2105, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/825fdbb3d277b26a5bcf7b540acba39e for a full test report.

**Comment by boegel** [comment]
@akesandgren Make sure you're using `develop`, you need https://github.com/easybuilders/easybuild-easyblocks/pull/2547 next to the custom easyblock for `jax`

**Comment by akesandgren** [comment]
I know, I just had forgotten to update...

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc060 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/45e24280e738bb4cd08bf9122f3e6aa1 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an04.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.5
See https://gist.github.com/731e7b7f01c735bad0c70360d384bf28 for a full test report.

File: easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.0.0-fosscuda-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+name = 'AlphaFold'
+version = '2.0.0'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+builddependencies = [
+    # CMake/oxygen/SWIG are required for building OpenMM
```
Comment (akesandgren) [inline easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.0.0-fosscuda-2020b.eb:12]: Doxygen, probably needs fixing in the non-GPU version too.

**Review by akesandgren — CHANGES_REQUESTED** [review]

**Review by akesandgren — APPROVED** [review]

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**